tests/coverage/coverage.cpp

```diff
@@ -80,16 +78,15 @@ TEST(Coverage, when)
     libcellml::When wm;
     libcellml::Reset r;
 
+    w.setValue("4738");
```

Comment:
`"4738"` ought to be declared as a string variable (and reused below in the `EXPECT_EQ()` statement).